frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/virtual-hardware-tab/virtual-hardware-tab.scss

```diff
@@ -0,0 +1,29 @@
+.kubevirt-create-vm-modal__virtual-hardware-tab-container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+
+  .kubevirt-create-vm-modal__virtual-hardware-tab-cd-title {
+    margin-top: var(--pf-global--spacer--md);
+  }
+
+  .kubevirt-create-vm-modal__virtual-hardware-tab-empty-state {
+    flex-direction: column;
+    align-items: start;
+    margin-top: var(--pf-global--spacer--sm);
+    padding-left: var(--pf-global--spacer--md);
```

Comment:
I think we can remove this padding now. When we expand on the Virtual hardware we can align with the latest designs for the review step where we add spacing to each section to separate rather than left padding. Better on mobile.  See here for what I mean.
https://codepen.io/matthewcarleton/pen/ExxMNbr?editors=1100